getting-started/setup-building.rst

```diff
@@ -245,47 +245,80 @@ still build properly).
 Windows
 -------
 
-For a quick guide to building you can read `this documentation`_ from Victor
-Stinner.
-
-All current versions of Python can be built using Microsoft Visual Studio 2017
-or later.  You can download
-and use any of the free or paid versions of `Visual Studio 2017`_.
-
-When installing Visual Studio 2017, select the **Python development** workload
-and the optional **Python native development tools** component to obtain all of
-the necessary build tools. If you do not already have git installed, you can
-find git for Windows on the **Individual components** tab of the installer.
+.. note:: If you are using the Windows Subsystem for Linux (WSL),
+   :ref:`clone the repository <checkout>` from a native Windows shell program
+   like PowerShell or the ``cmd.exe`` command prompt,
+   and use a build of Git targeted for Windows,
+   e.g. the `Git for Windows download from the official Git website`_.
+   Otherwise, Visual Studio will not be able to find all the project's files
+   and will fail the build.
+
+For a concise step by step summary of building Python on Windows,
+you can read `Victor Stinner's guide`_.
+
+All supported versions of Python can be built
+using Microsoft Visual Studio 2017 or later.
+You can download and use any of the free or paid versions of `Visual Studio`_.
+
+When installing it, select the :guilabel:`Python development` workload
+and the optional :guilabel:`Python native development tools` component
+to obtain all of the necessary build tools.
+You can find Git for Windows on the :guilabel:`Individual components` tab
+if you don't already have it installed.
 
 .. note:: If you want to build MSI installers, be aware that the build toolchain
-  for them has a dependency on the Microsoft .NET Framework Version 3.5 (which
-  may not be configured on recent versions of Windows, such as Windows 10). If
-  you are building on a recent Windows version, use the Control Panel (Programs
-  | Programs and Features | Turn Windows Features on or off) and ensure that the
-  entry ".NET Framework 3.5 (includes .NET 2.0 and 3.0)" is enabled.
+   for them has a dependency on the Microsoft .NET Framework Version 3.5
+   (which may not be included on recent versions of Windows, such as Windows 10).
+   If you are building on a recent Windows version, use the Control Panel
+   (:menuselection:`Programs --> Programs and Features --> Turn Windows Features on or off`)
+   and ensure that the entry
+   :guilabel:`.NET Framework 3.5 (includes .NET 2.0 and 3.0)` is enabled.
 
 Your first build should use the command line to ensure any external dependencies
 are downloaded:
 
-.. code-block:: dosbatch
+.. code-block:: batch
+
+   PCbuild\build.bat -c Debug
```

Comment:
Yeah; as mentioned above

> I considered [and went back and forth on] `-d`, but went with the former [`-c Debug`] to be more clear and explicit, and consistent with setting other build configs [as mentioned in the following paragraph]

but if you feel strongly otherwise, I can go back to `-d` (its what I would use for the quick step by step reference guide, at least)

Reply:
There aren't really any other interesting build configs. To do a PGO build, you'd pass `--pgo`, and you get a release build by omitting `-d` and `--pgo`. The fully customisable configuration is very legacy, though occasionally helpful for more intense debugging needs (i.e. not something the devguide is going to cover)